docs/source/repository-dir-layout.md

```diff
@@ -23,8 +25,16 @@
 
  libraries/                         # Libraries that are shared with all actors.
     common.py                       # These can be modules
-    sharedpkg/                      # or packages.
+    tests/                          # with tests stored here.
+        files/                      # If tests need to use some mocked files, they should be placed here
+                                    # and refenreced from the tests using 'relative path, i.e. 'tests/files'
+        test_common.py
+    sharedpkg/                      # Or they can be packages
         __init__.py
+        tests/                      # with tests stored here.
+            test_sharedpkg.py
+            files/                  # If tests need to use some mocked files, they should be placed here
+                                    # and refenreced from the tests using 'absolute path' 'sharedpkg/tests/files'
```

Comment:
```suggestion
                                    # and refenreced from the tests using 'relative path' 'sharedpkg/tests/files'
```

Reply:
This should really be 'absolute' path (note the quotation marks). Absolute in the meaning that `sharedpkg` needs to be included in the path, as opposed to 'reative' path (used e.g. in actors), where `tests/files` is used.

Reply:
well, `sharepkg` doesn't really make it absolute...

but I will let others decide

Reply:
and what about just using word "path"? Just suggestion to move it forward :)

E.g.:
`and refenreced from the tests using 'sharedpkg/tests/files' path?`

Just a small nit: there seems to be a typo in `refenreced`

Reply:
Fixed